### PR TITLE
Reverted calling VS batch api while executing nuget actions with packages.config

### PR DESCRIPTION
**Pull request by jainaashish**
Fixes https://github.com/NuGet/Home/issues/3239 

Since calling VS batch api to improve perf while updating packages, it delayed updating project references until all packages are updated which caused a regression in adding binding redirects. Because of this change, references were not in correct state while adding binding redirects in files like web.config So removed these batch api for now.

@rrelyea @emgarten @drewgil @joelverhagen

### CODE REVIEW

**Comment by emgarten** [comment]
:shipit:

File: test/EndToEnd/tests/update.ps1 [hunk 1]
```diff
@@ -1737,4 +1737,27 @@ function Test-UpdatePackageWithToHighestMinorFlag {
...
+function Test-UpdatingBindingRedirectAfterUpdate {
+    param(
+        $context
+    )
+
+    # Arrange
+    $p = New-WebApplication
+
+    # Act
+	$p | Install-Package B -Version 2.0 -Source $context.RepositoryPath
```
Comment (joelverhagen) [inline test/EndToEnd/tests/update.ps1]: nit: consistent indentation

File: test/EndToEnd/tests/update.ps1 [hunk 2]
```diff
@@ -1737,4 +1737,27 @@ function Test-UpdatePackageWithToHighestMinorFlag {
     Assert-Null (Get-ProjectPackage $p A 1.0.0)
     Assert-Null (Get-ProjectPackage $p B 1.0.0)
     Assert-Null (Get-ProjectPackage $p C 1.0.0)
+}
+
+function Test-UpdatingBindingRedirectAfterUpdate {
```
Comment (joelverhagen) [inline test/EndToEnd/tests/update.ps1:1742]: Sweeeeeeet! :+1:

**Comment by joelverhagen** [comment]
:shipit:

**Comment by jainaashish** [comment]
Merged 6d5f90b0be22372be25fa117a87038ae06a71227

**Pull request closed without merging**